cluster/gce/util.sh

```diff
@@ -112,7 +112,10 @@ if [[ "${ENABLE_CLUSTER_AUTOSCALER}" == "true" ]]; then
   fi
 fi
 
+# These prefixes must not be prefixes of each other, so that they can be used to
+# detect mutually exclusive sets of nodes.
 NODE_INSTANCE_PREFIX=${NODE_INSTANCE_PREFIX:-"${INSTANCE_PREFIX}-minion"}
+WINDOWS_NODE_INSTANCE_PREFIX=${WINDOWS_NODE_INSTANCE_PREFIX:-"${INSTANCE_PREFIX}-windows-node"}
```

Comment:
Since `windows-node` is longer than `minion` I guess we should make sure that we won't exceed any length limits in node names or group names after this change?

Reply:
True... but on the other hand, the current approach for windows node names is to just insert windows after the prefix, e.g. "e2e-121-36623-minion-windows-group-8px9". So with this change I'm shortening what's already there (minion -> node) :)

I'm not totally sure where to look for places this would matter, so I say we fix things if/when we find they're broken.

https://cloud.google.com/compute/docs/reference/rest/v1/instances says that names can be up to 63 characters, fyi. Not sure about instance group names.